bump-exclude.yaml

```diff
@@ -0,0 +1,3 @@
+# Don't have ./hack/update-deps.sh
+- "google/exposure-notifications-server"
+- "tektoncd/triggers"
```

Comment:
I'd disable this for (at least) all of the Tekton repos.

Reply:
They are removing all repos: https://github.com/knative-sandbox/knobots/pull/20